src/utils/log.ts

```diff
@@ -23,17 +29,20 @@ export enum LogType {
   commandParser = 'Command parser',
 }
 
-export function log(logType: LogType): (...message: unknown[]) => void {
-  return (...messages: unknown[]) => {
-    console.log(logType, ...messages);
-    // Add messages to the Mapper Tab Page, if exists.
-    // Dynamic lookup to avoid circlular dependency.
-    if ('MapperTabPage' in globalThis) {
-      (
-        globalThis as unknown as {
-          MapperTabPage: {log: (...message: unknown[]) => void};
-        }
-      )['MapperTabPage'].log(logType, ...messages);
-    }
-  };
+/* eslint "@typescript-eslint/no-extraneous-class": "off" */
+export class MapperTabPage {
```

Comment:
`MapperPageTab` is not a thing in case of Puppeteer, so I'd prefer to keep it more abstract